scripts/system/libraries/gridTool.js

```diff
@@ -163,6 +163,14 @@ Grid = function() {
         newPosition = Vec3.subtract(newPosition, { x: 0, y: SelectionManager.worldDimensions.y * 0.5, z: 0 });
         that.setPosition(newPosition);
     };
+    
+    that.moveToAvatar = function() {
+        var position = MyAvatar.getJointPosition("LeftFoot");
+        if (position.x === 0 && position.y === 0 && position.z === 0) {
```

Comment:
Better to use `0.0`s, I think, to indicate that they're real numbers rather than integeters.